packages/block-serialization-default-parser/parser.php

```diff
@@ -63,10 +63,10 @@ class WP_Block_Parser_Block {
 	public $innerContent;
 
 	function __construct( $name, $attrs, $innerBlocks, $innerHTML, $innerContent ) {
-		$this->blockName   = $name;
-		$this->attrs       = $attrs;
-		$this->innerBlocks = $innerBlocks;
-		$this->innerHTML   = $innerHTML;
+		$this->blockName    = $name;
+		$this->attrs        = $attrs;
+		$this->innerBlocks  = $innerBlocks;
+		$this->innerHTML    = $innerHTML;
```

Comment:
Do you know why this wasn't caught by the phpcs config in Gutenberg? should we update it?

Reply:
🤔 Good question. Let me dig in a bit. I am not familiar with the plugin's ruleset.

Reply:
Hmm, now that I look, there are a number of PHPCS issues in this file. Wonder if it's running against it at all.

Reply:
There is `<exclude-pattern>./lib/parser.php</exclude-pattern>` in the PHPCS file, but I don't think this is excluding this specific file.

I think that the file is just not being processed from lack of inclusion rather than exclusion. Replacing all the `<file></file>` tags and replacing them with just `<file>.</file>` (all files) correctly formats this file, but also a handful of others (but all other files are e2e or phpunit test related).

If possible, I'd like to merge this change, and explore modifying the ruleset in a separate PR.